src/styles/blocks/clock__center.scss

```diff
@@ -0,0 +1,8 @@
+.clock__center {
+  @extend %position;
+  z-index: 2;
+  transform: translate(-40%, 50%);
+  @include circle($circle-size);
+  border-radius: 50%;
+  background-color: $main-color;
+}

```

Comment:
place all elements to the clock file. elements should be placed in the same file as the block. And use nesting